src/sokoban.hpp

```diff
@@ -1,22 +1,27 @@
 #ifndef __SOKOBAN_H__
 #define __SOKOBAN_H__
 
+#include <iostream>
+#include <stdexcept>
```

Comment:
I believe the best practice is to move the include to the .cpp file since the .hpp doesn't need/use it.